src/java.base/share/classes/sun/net/www/protocol/http/AuthCacheValue.java

```diff
@@ -35,24 +34,17 @@
  * @author Michael McMahon
  */
 
-public abstract class AuthCacheValue implements Serializable {
-
-    @java.io.Serial
-    static final long serialVersionUID = 735249334068211611L;
+public abstract class AuthCacheValue {
 
     public enum Type {
         Proxy,
         Server
     };
 
     /**
-     * Caches authentication info entered by user.  See cacheKey()
+     * The default authentication cache
      */
-    protected static AuthCache cache = new AuthCacheImpl();
-
-    public static void setAuthCache (AuthCache map) {
-        cache = map;
-    }
+    protected static final AuthCacheImpl defCache = new AuthCacheImpl();
```

Comment:
Is this still needed?

Reply:
Right, that is redundant now as the default instance is created in `AuthCacheImpl`. I will remove it.